### ISSUE TITLE
Execute bulk tx with ledger fails the gas estimation

### ISSUE THREAD

**Issue by francovenica** [safe-global/safe-wallet-web#2561]
## Bug description
Trying to execute a tx in bulk with ledger fails. This doesn't happen with other wallets.

## Environment
 - Browser: Chrome
 - Wallet: Ledger
 - Chain: Goerli
 - Env: Current prod 1.18
 - Device: Ledger Nano S

## Steps to reproduce
 1. Go to a safe and queue 2 tx fully signed
 2. Connect with a ledger owner
 3. Go to the "execute in bulk" form
 4. Click "Submit"

## Expected result
In the device the prompt to sign is shown

## Obtained result
An error shows

The error:
`cannot estimate gas; transaction may fail or may require manual gas limit [ See: https://links.ethers.org/v5-errors-UNPREDICTABLE_GAS_LIMIT ] (error={"code":-32000,"message":"gas required exceeds allowance (31059)"}, method="estimateGas", transaction={"from":"0x6f965E48347AF3Df65c14CCc176A9CbeCEa0eDb5","maxPriorityFeePerGas":{"type":"BigNumber","hex":"0x59682f00"},"maxFeePerGas":{"type":"BigNumber","hex":"0x37e11d6000"},"to":"0x40A2aCCbd92BCA938b02010E17A5b8929b49130D","data":"0x8d80ff0a000000000000000000000000000000000000000000000000000000000000002000000000000000000000000000000000000000000000000000000000000005f20000818a0babbf784f89055ebca196d76bd8a2a966000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000002e46a76120200000000000000000000000000818a0babbf784f89055ebca196d76bd8a2a9660000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000014000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000001e00000000000000000000000000000000000000000000000000000000000000064e318b52b000000000000000000000000000000000000000000000000000000000000000100000000000000000000000061a0c717d18232711bc788f19c9cd56a43cc88720000000000000000000000007724b234c9099c205f03b458944942bceba134080000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000c38e2df212f12f67da83c192c73a1e3e00671fea79d8a66476886239f17a79fc2f05be5a5366234d5b6f50eb83a6dfcad9a8dfbf856164428991e7b7ff75c635e41bb578cabfaeba36a8969d812f12cdc851f8f14370737aab2023b5897c7258a9986495f066739acadab9c92059d383b74bbdb550e8b1aee18d1c341280c4b3422f1f0d166b66ce3e911bd725f39fe04c70284993aac5d9c2f4cfd44a0168c5c0957940668dc47b438fec403392c90cee2c4202d68d24b73131f66bc18d71cf178d361f00000000000000000000000000000000000000000000000000000000000000818a0babbf784f89055ebca196d76bd8a2a966000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000002646a76120200000000000000000000000000818a0babbf784f89055ebca196d76bd8a2a966000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000001400000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000160000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000c397fdcc1df15822126eb4ba0ce0f0fb2c8b4551215baa97fbf5115bb15f30ab22171dfe0b32cc977be43e940487e689071aa96e371b81c909f98daf73907d81fc206cd53dcd396560d24354c34c96b383485e5a93fb8ecd6d6d3121aefa5334d4da737cdbec84ab73b01cf50536fac022af253b24e461e905d85d4cb371191536d6200d35f898281cc3a95df0e31cf1906b3d5b2a2f9a7a537e352d4b1d0cd264689b053799e276c3ee8ead6f93ba80efd3cee0588fd99ed72aeb5039a78fc89a1d9c2000000000000000000000000000000000000000000000000000000000000000000000000000000000000000","accessList":null}, code=UNPREDICTABLE_GAS_LIMIT, version=providers/5.7.2)`

## Screenshots
![image](https://github.com/safe-global/safe-wallet-web/assets/4503659/31f56f01-7c23-4c0c-8ecb-55cc20f5b44a)

**Comment by francovenica** [comment]
So I tried with a Ledger X and I didn't have any issues.

I tried with a Ledger S and I had a different error this time.
I was trying to execute 3 tx (2 send funds and adding an owner).
This is the safe I was trying to use: 
https://safe-wallet-web.staging.5afe.dev/transactions/queue?safe=sep:0x98bC720d9C91474e8B2EbE9E08d4aBdb043a2d7a
![image](https://github.com/safe-global/safe-wallet-web/assets/4503659/cd690e5b-e4a0-47fa-849a-ae06414643cb)
